app/components/Fonticon.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+const Fonticon = (props) => {
+    const {
+      className
+    } = props;
+
+    return (
+        <i className ={className} ></i>
+    );
+
+}
+
+export default Fonticon;
```

Comment:
1. Always put a newline in the end.
2. Unused import `Component`.
3. No types defined for component and props